webhook/notifier.go

```diff
@@ -34,16 +34,30 @@ func NewDefaultNotifier(apiKey, apiSecret string, urls []string) QueuedNotifier
 	n := &DefaultNotifier{}
 	for _, url := range urls {
 		u := NewURLNotifier(URLNotifierParams{
-			URL:       url,
-			Logger:    logger.GetLogger().WithComponent("webhook"),
-			APIKey:    apiKey,
-			APISecret: apiSecret,
+			URL:          url,
+			Logger:       logger.GetLogger().WithComponent("webhook"),
+			APIKey:       apiKey,
+			APISecret:    apiSecret,
+			DropWhenFull: true,
 		})
 		n.urlNotifiers = append(n.urlNotifiers, u)
 	}
 	return n
 }
 
+func NewDefaultNotifierByParams(params []URLNotifierParams) QueuedNotifier {
```

Comment:
For more control. With this we can have different configuration per webhook. But the real reason was to avoid long constructors. It was simpler just to pass in a list of `URLNotifierParams`.